uamqp/async_ops/connection_async.py

```diff
@@ -108,7 +108,7 @@ def loop(self):
         return self._internal_kwargs.get("loop")
 
     async def lock_async(self, timeout=3.0):
-        await asyncio.wait_for(self._async_lock.acquire(), timeout=timeout, **self._internal_kwargs)
+        await asyncio.wait_for(await self._async_lock.acquire(), timeout=timeout, **self._internal_kwargs)
```

Comment:
I think this would actually give an error?

depending on [the doc of asyncio.wait_for](https://docs.python.org/3/library/asyncio-task.html#asyncio.wait_for), it's taking an await object.

after adding this await, we turn await object into an unawait object?

Reply:
hmm yeah that's strange, I looked up the usage of .wait_for up and this isn't it. The original should work. I'll keep looking.